homework/index.js

```diff
@@ -30,17 +32,152 @@
+      text: label,
+      class: 'table-header',
     });
+    createAndAppend('td', row, {
+      text: content,
+    });
+    return row;
+  }
+
+  function renderRepoInfo(repository) {
+    const leftSide = document.getElementById('left-side');
+    leftSide.innerHTML = '';
+    const repoTable = createAndAppend('table', leftSide);
+    const repoTableBody = createAndAppend('tbody', repoTable);
+    const repoNameRow = addRepoInfoRow(repoTableBody, 'Repository', '');
+    createAndAppend('a', repoNameRow.lastChild, {
+      href: repository.html_url,
+      target: '_blank',
+      text: repository.name,
+    });
+    addRepoInfoRow(repoTableBody, 'Description', repository.description);
+    addRepoInfoRow(
+      repoTableBody,
+      'Created At',
+      new Date(repository.created_at).toLocaleString('en-GB'),
+    );
+    addRepoInfoRow(
+      repoTableBody,
+      'Updated At',
+      new Date(repository.updated_at).toLocaleString('en-GB'),
+    );
+    addRepoInfoRow(repoTableBody, 'Forks', repository.forks_count);
+    addRepoInfoRow(repoTableBody, 'Watchers', repository.watchers_count);
   }
 
-  const HYF_REPOS_URL = 'https://api.github.com/orgs/HackYourFuture/repos?per_page=100';
-  window.onload = () => main(HYF_REPOS_URL);
+  function fetchAndRender(repository) {
+    const rightSide = document.getElementById('right-side');
+    rightSide.innerHTML = '';
+    fetchJSON(repository.contributors_url)
+      .then(contributors => {
+        const contributorsTitle = createAndAppend('h3', rightSide, {
+          text: 'Contributors',
+          class: 'contributors-title',
+        });
+        if (!contributors.length) {
+          contributorsTitle.textContent = 'No Contributor So Far';
+          return;
+        }
+        const contributorsList = createAndAppend('ul', rightSide, {
+          class: 'contributors-list',
+        });
+        contributors.forEach(contributor => {
+          const listItem = createAndAppend('li', contributorsList, {
+            class: 'contributor',
+            tabindex: 0,
+            'aria-label': contributor.login,
+          });
+          createAndAppend('img', listItem, {
+            class: 'contributor-avatar',
+            src: contributor.avatar_url,
+          });
+          createAndAppend('span', listItem, {
+            class: 'contributor-name',
+            text: contributor.login,
+          });
+          createAndAppend('span', listItem, {
+            class: 'contribution-count',
+            text: contributor.contributions,
+          });
+          listItem.addEventListener('click', () => {
+            // Go to new page on click
+            window.open(contributor.html_url, '_blank');
+          });
+          listItem.addEventListener('keyup', event => {
+            // Open new page with contributor url when pressed Enter on focused contributor
+            if (event.key === 'Enter') window.open(contributor.html_url, '_blank');
+          });
```

Comment:
Adding these event listeners in combination with setting the tabindex allows the list item to be activated with both the mouse and the keyboard. Alternatively, you could use an `a` tag that implements this behaviour by default.